components/tidb_query/src/util.rs

```diff
@@ -4,6 +4,9 @@ use kvproto::coprocessor as coppb;
 use tipb::ColumnInfo;
 
 use crate::codec::datum::Datum;
+use rand::SeedableRng;
+use rand_xorshift::XorShiftRng;
+use std::cell::RefCell;
```

Comment:
Prefer to put `std` import at the very top. Before `use tipb::ColumnInfo;` in this case, with a blank line between.

Reply:
remove RefCell use.